core/src/main/scala/dagr/core/tasksystem/Dependable.scala

```diff
@@ -29,14 +29,52 @@ package dagr.core.tasksystem
   */
 trait Dependable {
   /** Creates a dependency on this dependable, for the provided Task. */
-  def ==> (other: Task) : other.type
+  def ==> (other: Dependable) : Dependable
 
-  /** Creates a dependency on this dependable for all the Tasks in the provided MultiTask. */
-  def ==> (others: MultiTask) : MultiTask
+  /** Breaks the dependency link between this dependable and the provided Task. */
+  def !=> (other: Dependable) : Unit
+
+  /** Returns an object that can be used to manage dependencies that apply to this and the other Dependable. */
+  def :: (other: Dependable) : Dependable = new DependencyGroup(this, other)
```

Comment:
Can you point me to the test that uses `Dependable.::` versus `Multitask.::`?  I am confused about which to use and when, or if I need to understand that?